shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -37,7 +49,99 @@ void PlatformChannel::HandleMethodCall(
   } else if (method == "HapticFeedback.vibrate") {
     result->NotImplemented();
   } else if (method == "Clipboard.getData") {
-    result->NotImplemented();
+    const rapidjson::Value& format = call.arguments()[0];
```

Comment:
Consider moving the body of this "if" to a separate method to make `PlatformChannel::HandleMethodCall` shorter.